database/rpcdb/db_client.go

```diff
@@ -189,16 +186,85 @@ type iterator struct {
 	db *DatabaseClient
 	id uint64
 
-	data []*rpcdbpb.PutRequest
-	errs wrappers.Errs
+	data        []*rpcdbpb.PutRequest
+	fetchedData chan []*rpcdbpb.PutRequest
```

Comment:
Should we use a buffered channel here (of say 5) instead to prefetch a bit more in the case that the caller processes data more quickly than the time it takes for a get to occur?